### DISCUSSION TITLE
Messenger App based on Bluetooth Connectivity to bypass internet outage in protests

### DISCUSSION THREAD

**Discussion by anonymoper** [WeAreMahsaAmini/Core#66] [Ideas]
Just suppose bypassing internet outage and using a Bluetooth messenger app for the communication and coordination of protests. The app must be designed for this purpose so that person A can send a message to B, then B to C, ... to Z :). Problem solved? With the evolution of Bluetooth hardware, we can get a good enough distance for the connection.

LMK what you think

Open to enhancing the idea

**Comment by norsnap** [comment]
Bluetooth messaging apps like [bridgefy](https://bridgefy.me/) exists and was used by [Hong Kong protesters](https://www.bbc.com/news/technology-49565587).
We just need to promote the use of it.

**Reply by anonymoper** [reply]
@aboutsajjad  Nice! Bridgefy has no Persian version. We can ask them to translate their app and promote it, or we use their SDK to create a customized version of the messenger.

**Comment by AnonymousMaximus2** [comment]
and also this idea can complement your idea:

https://github.com/WeAreMahsaAmini/Core/discussions/115

**Comment by bardiakeyvani** [comment]
There is also [Briar](https://briarproject.org/how-it-works/), uses Bluetooth, Wifi, and Tor simultaneously. The only downfall is it is Android only.
